website/docs/r/team_membership.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
one more thing, 
can you add this to opsgenie.erb file as well terraform website shows urls based on that file

Reply:
ouch. thx! will change